third-party/crowdsec/sample-configuration/collections/james.yaml

```diff
@@ -2,5 +2,5 @@ parsers:
   - crowdsecurity/syslog-logs
   - linagora/james-connect-security-log
 scenarios:
-  - linagora/james-bf-auth
+  - /apache-james/bf-auth
```

Comment:
bf-auth -> brute-force-auth ?

Reply:
I see that there are many collections in [Crowdsec Hub](https://app.crowdsec.net/hub/collections) use `bf` so I think we should `bf`.

Reply:
Ok